### DISCUSSION TITLE
Good ideas to add in a future update?

### DISCUSSION THREAD

**Discussion by Shzv** [diamond-rip/Eden#86] [Ideas]
these are some suggestions that i think are useful for eden i hope you add them

• the possibility to modify the menus as you want (ranked,unranked)
• edit the tablist (header,footer)
• add rank with permissions via eden.
(VIP, staff ..)

**Comment by GoodestEnglish** [comment]
Rank will not be added since Eden is a practice core, not a server core. You should consider use a seperate permission plugin like LuckPerms if you want to add ranks.

Will add the others soon.

**Reply by Shzv** [reply]
I asked this being that luckperms is not implemented well being that it causes bugs in tablist and in chat and in game it rewrites opponent and you colors https://i.imgur.com/MxsPy4x.png

**Comment by GoodestEnglish** [comment]
Added tablist header footer config in https://github.com/diamond-rip/Eden/releases/tag/1.0.27-BETA-cd98a9a. Menu modify is slowly being developed
